trie/trieNodesHandler.go

```diff
@@ -0,0 +1,63 @@
+package trie
+
+type trieNodesHandler struct {
```

Comment:
Maybe it would help the debugging process to print the size of these maps/array from time to time.

Reply:
Will leave a todo to be refactored when integrating it into rc/v1.4.0
I have to use the statistics component since we have ~200 instances running in the same time